.github/workflows/gen-completion.yml

```diff
@@ -0,0 +1,27 @@
+name: Generate ohmyzsh completion
+on:
+  push:
+    branches:
```

Comment:
Does this then update the script for every push to main/release branches? My preference would be to do this manually (in the release checklist) or add it to the release task

Reply:
@mmulji-ic have removed the automation script so as to allow for manual generation and distribution of the script rather than relying on automation.